packages/actors_api/src/actors/Attester.ts

```diff
@@ -92,12 +93,46 @@ export async function issueAttestation(
  *
  * @param attester The [[AttesterIdentity]] which signed the [[Attestation]] in [[issueAttestation]].
  * @param revocationHandle A reference to the [[Attestation]] which was created in [[issueAttestation]].
+ * @throws [[ERROR_UNAUTHORIZED]], [[ERROR_NOT_FOUND]].
  */
 export async function revokeAttestation(
   attester: Identity,
   revocationHandle: IRevocationHandle
 ): Promise<void> {
-  await Attestation.revoke(revocationHandle.claimHash, attester).then((tx) =>
-    BlockchainUtils.submitTxWithReSign(tx, attester)
-  )
+  const attestation = await Attestation.query(revocationHandle.claimHash)
+
+  if (attestation === null) {
+    throw SDKErrors.ERROR_NOT_FOUND('Attestation not on chain')
+  }
```

Comment:
mmh that means we were submitting a transaction before with an extrinsic that's doomed to fail? Guess this is more efficient now :D